include/tvm/auto_scheduler/transform_step.h

```diff
@@ -118,6 +123,8 @@ class IteratorNode : public Object {
   IteratorKind iter_kind;
   /*! \brief The annotation type of this iterator. */
   IteratorAnnotation annotation;
+  /*! The original iterators before fusion. */
+  std::vector<Iterator> ori_iters;
```

Comment:
Is there any better way to store the original iterators? Maybe directly get from the ComputeDAG's init_state or original op?
It seems strange to have a Iterator list inside a IteratorNode object.